src/main/java/it/pagopa/pm/gateway/config/ClientConfig.java

```diff
@@ -36,27 +38,46 @@ public class ClientConfig {
 
     private static final String HTTPS_PROXY_HOST_PROPERTY = "https.proxyHost";
     private static final String HTTPS_PROXY_PORT_PROPERTY = "https.proxyPort";
-    @Value("${bancomatPay.client.url}")
-    private String BPAY_CLIENT_URL;
-
-    @Value("${bancomatPay.client.timeout.ms:5000}")
-    private int BPAY_CLIENT_TIMEOUT_MS;
-
-    @Value("${azureAuth.client.maxTotal:100}")
-    private int AZURE_CLIENT_MAX_TOTAL;
-
-    @Value("${azureAuth.client.maxPerRoute:100}")
-    private int AZURE_CLIENT_MAX_PER_ROUTE;
-
-    @Value("${azureAuth.client.timeout.ms:5000}")
-    private int AZURE_CLIENT_TIMEOUT;
 
     @Value("${postepay.client.url}")
     private String POSTEPAY_CLIENT_URL;
 
     @Value("${postepay.client.timeout.ms:5000}")
     private int POSTEPAY_CLIENT_TIMEOUT;
 
+    @Value("${azureAuth.client.config")
+    private String AZUREAUTH_CLIENT_CONFIG;
+
+    @Value("${bancomatPay.client.config}")
+    private String BANCOMAT_CLIENT_CONFIG;
+
+    private final Map<String, String> azureAuthClientConfigValues;
+    private final Map<String, String> bancomatClientConfigValues;
+
+    {
+        azureAuthClientConfigValues = getAzureAuthClientConfigValues();
+        bancomatClientConfigValues = getBancomatClientConfigValues();
+    }
+
+    private Map<String, String> getAzureAuthClientConfigValues() {
+        List<String> listConfig = Arrays.asList(AZUREAUTH_CLIENT_CONFIG.split(PIPE_SPLIT_CHAR));
+        Map<String, String> configsMap = new HashMap<>();
+        configsMap.put(MAX_TOTAL, listConfig.get(0));
+        configsMap.put(MAX_PER_ROUTE, listConfig.get(1));
+        configsMap.put(TIMEOUT_MS, listConfig.get(2));
+
+        return configsMap;
+    }
+
+    private Map<String, String> getBancomatClientConfigValues() {
+        List<String> listConfig = Arrays.asList(BANCOMAT_CLIENT_CONFIG.split(PIPE_SPLIT_CHAR));
+        Map<String, String> configsMap = new HashMap<>();
+        configsMap.put(URL, listConfig.get(4));
```

Comment:
Maybe it's better if you call this property BPAY_URL since it is the URL for Bancomat Pay